data/transactions/logic/eval_test.go

```diff
@@ -3532,8 +3532,7 @@ func benchmarkBasicProgram(b *testing.B, source string) {
 func benchmarkOperation(b *testing.B, prefix string, operation string, suffix string) {
 	runs := 1 + b.N/2000
 	inst := strings.Count(operation, ";") + strings.Count(operation, "\n")
-	source := prefix + ";" + strings.Repeat(operation+";", 2000) + ";" + suffix
-	source = strings.ReplaceAll(source, ";", "\n")
+	source := prefix + ";" + strings.Repeat(operation+"\n", 2000) + ";" + suffix
```

Comment:
Is this just to make assembly take less time or was there an error?

Reply:
Turned out to be necessary. `bufio.Scanner` has a `MaxTokenSize`. Since we ask it to read entire lines, a "token" is an entire line. So, as I removed substitutions of "\n" for ";", our benchmarks had enormous lines that exceeded that limit. My adding a newline in the main repeated clause, we avoid the problem. (And, I inserted some code to check `scanner.Err()` in case we exceed it in the future. (And to report the error as "too long line" when it happens.)